static/js/components/CreateCommentForm_test.js

```diff
@@ -98,7 +98,8 @@ describe("CreateCommentForm", () => {
     })
 
     it("should begin form editing on load", async () => {
-      wrapper = await renderPostForm()
+      wrapper = renderPostForm()
+      // $FlowFixMe
```

Comment:
Could you add a description to this so we know what flow error is underlying this?